frontend/src/index.js

```diff
@@ -14,8 +15,10 @@ import {ScheduleContextProvider} from './context/ScheduleContext';
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
+    <ClinicContextProvider>
     <ScheduleContextProvider>
```

Comment:
Here it should be tabbed. Nest these context providers for now (tab after every context like HTML Structure). It will be fixed in the later versions.